lib/pages/dashboard_page.dart

```diff
@@ -94,9 +110,14 @@ class _DashboardPageState extends State<DashboardPage> {
                     onTap: () {
                       setState(() {
                         selectedItem = 3;
+                        selectedItemList[0] = false;
+                        selectedItemList[1] = false;
+                        selectedItemList[2] = false;
+                        selectedItemList[3] = true;
```

Comment:
There is so much repetition going on here. Create a function that does this for you and returns this selectedItemList.

Reply:
@ahamzatariq , Please fix this before i merge this.

Reply:
@waleedwaseem implemented. In the last commit of the same PR .aka. PR#1